src/NServiceBus.AcceptanceTests/Scheduling/When_scheduling_a_recurring_task.cs

```diff
@@ -22,7 +22,7 @@ await Scenario.Define<Context>()
                         Assert.True(c.InvokedAt.HasValue);
                         Assert.Greater(c.InvokedAt.Value - c.RequestedAt, TimeSpan.FromMilliseconds(5));
                     })
-                  .Run(TimeSpan.FromSeconds(20));
+                  .Run(TimeSpan.FromSeconds(60));
```

Comment:
Should we let the transports affect this via the `ConfigureXYZTransport` class in some way?